packages/database/src/migrations/20210607003511-MigrateCurrentDashboardsToNewDashboards-modifies-data.js

```diff
@@ -0,0 +1,332 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const selectUniqueDashboardGroupNameAndOrgUnitCombo = async db => {
+  const dashboardGroups = await db.runSql(`
```

Comment:
Not a big deal since migrations are one and done code, but these first two getter functions are inconsistent with the rest

```suggestion
  const { rows: dashboardGroups } = await db.runSql(`
```